diff --git a/settings/defaults.go b/settings/defaults.go
index 471e368..77dd019 100644
--- a/settings/defaults.go
+++ b/settings/defaults.go
@@ -55,9 +55,9 @@ func defaultSettings() Settings {
 			"SNK - Neo Geo Pocket":                           "mednafen_ngp_libretro",
 			"Sony - PlayStation":                             playstationCore,
 		},
-		CoresDirectory:       "./cores",
-		AssetsDirectory:      "./assets",
-		DatabaseDirectory:    "./database",
+		CoresDirectory:       "@out@/share/ludo/cores",
+		AssetsDirectory:      "@out@/share/ludo/assets",
+		DatabaseDirectory:    "@out@/share/ludo/database",
 		SavestatesDirectory:  filepath.Join(usr.HomeDir, ".ludo", "savestates"),
 		SavefilesDirectory:   filepath.Join(usr.HomeDir, ".ludo", "savefiles"),
 		ScreenshotsDirectory: filepath.Join(usr.HomeDir, ".ludo", "screenshots"),
diff --git a/settings/settings.go b/settings/settings.go
index e9c3d58..4589e3d 100644
--- a/settings/settings.go
+++ b/settings/settings.go
@@ -71,9 +71,9 @@ func Load() error {
 	// Set default values for settings
 	Current = Defaults
 
-	// If /etc/ludo.json exists, override the defaults
-	if _, err := os.Stat("/etc/ludo.json"); !os.IsNotExist(err) {
-		b, _ := ioutil.ReadFile("/etc/ludo.json")
+	// If @out@/etc/ludo.json exists, override the defaults
+	if _, err := os.Stat("@out@/etc/ludo.json"); !os.IsNotExist(err) {
+		b, _ := ioutil.ReadFile("@out@/etc/ludo.json")
 		err = json.Unmarshal(b, &Current)
 		if err != nil {
 			return err
